src/Core/Checkout/Order/Aggregate/OrderTransaction/OrderTransactionStateHandler.php

```diff
@@ -112,11 +113,28 @@ public function paid(string $transactionId, Context $context): void
     }
 
     /**
+     * @deprecated tag:v6.8.0 - Will be removed. Use OrderTransactionStateHandler::paidPartially instead
+     *
      * @throws InconsistentCriteriaIdsException
      * @throws StateMachineException
      * @throws IllegalTransitionException
      */
     public function payPartially(string $transactionId, Context $context): void
+    {
+        Feature::triggerDeprecationOrThrow(
+            'v6.8.0.0',
+            Feature::deprecatedMethodMessage(__CLASS__, __METHOD__, 'v6.8.0.0', 'OrderTransactionStateHandler::payPartially')
```

Comment:
Correct replacement:
```suggestion
            Feature::deprecatedMethodMessage(__CLASS__, __METHOD__, 'v6.8.0.0', __CLASS__ . '::paidPartially')
```